docs/gathered-data.md

```diff
@@ -52,17 +52,32 @@ The CRD sizes above are in the raw (uncompressed) state.
 
 ## CephCluster
 
-collects statuses of the`cephclusters.ceph.rook.io` resources
+Collects statuses of the`cephclusters.ceph.rook.io` resources
 from Openshift Data Foundation Stack.
 
-API Reference:
+### API Reference:
+https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
 
-	https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go
+### Sample data:
+docs/insights-archive-sample/config/storage/openshift-storage/cephclusters/ocs-storagecluster-cephcluster.json
 
-* Location in archive: config/storage/<namespace>/<name>.json
-* Id in config: clusterconfig/ceph_cluster
-* Since versions:
-  - 4.12+
+### Location in archive:
+| Version | Path													 |
+| ------- | ----------------------------------------------------- |
+| <= 4.12 | config/storage/{namespace}/{name}.json 				 |
+| 4.13+   | config/storage/{namespace}/cephclusters/{name}.json 	 |
+
+### Config ID:
+clusterconfig/ceph_cluster
+
+### Released version:
+* 4.12
+
+### Backport versions:
+* 4.8.49
+* 4.9.48
+* 4.10.31
+* 4.11.2
```

Comment:
So why can't we keep the original `Since versions` format? We will have two gatherers docs with different format from the rest. I personally can't see any difference between released version and backport versions.